src/components/Modal/Modal.js

```diff
@@ -111,13 +112,12 @@ class Modal extends Component {
       hasCloseButton,
       ...props
     } = this.props
-    const classNames = this.className
-    console.log(classNames)
     return (
       <div className='sprk-u-JavaScript'>
         <div data-sprk-modal-trigger={id} />  {/* This needs to be fixed, their method looks for this attribute and sets focus to it */}
         <div ref={this.mainRef}>
           <div
+            {...props}
```

Comment:
I think this should go on line 131 for consistency. Maybe the reason it didn't work was because of the `className` on line 50.

Reply:
these changes didn't fix the issue unfortunately.

Reply:
`className` should still be destructured from `this.props` on line ~96.